src/map.h

```diff
@@ -344,4 +341,6 @@ class MMVManip : public VoxelManipulator
 		value = flags describing the block
 	*/
 	std::map<v3s16, u8> m_loaded_blocks;
+
+	static constexpr u8 VMANIP_BLOCK_DATA_INEXIST = 1;
```

Comment:
I think that an enum (despite being 1 entry) would be more intuitive. Same approach as seen in `VOXELFLAG_NO_DATA`.

```suggestion
	enum : u8 {
		VMANIP_BLOCK_DATA_INEXIST = 1 << 0,
	};
```